rpc/api.proto

```diff
@@ -1054,6 +1054,8 @@ message DecodedTransaction {
 			STAKE_SUB_CHANGE = 9;
 			PUB_KEY_ALT = 10;
 			PUB_KEY_HASH_ALT = 11;
+			TGEN = 12;
+			TADD = 13;
```

Comment:
Changes to this file may require an update for Decrediton at https://github.com/decred/decrediton/commits/master/app/middleware/api.proto  Not sure how the Decrediton interaction works

Reply:
Requiring updates when api.proto is changed is true in general, but I don't think we're using these in Decrediton. In any case, we'll update our version as needed.